### DISCUSSION TITLE
Updating Help

### DISCUSSION THREAD

**Discussion by sohail579** [osy86/HaC-Mini#1037] [Q&A] [answered]
If I want to update to Ventura do I just use software update again? and once its updated rerun the patcher?

**Comment by osy** [comment, marked as answer]
You only need to patch once. If you’ve enabled secure boot, software updates should work as is. Follow the guide for complete details.

**Reply by sohail579** [reply]
Thanks i had everything set up and wanted to update to ventura so ran software update it then on the boot screen has the installer but keeps looping what am I doing wrong
